dluhc-component-library/src/components/policyForm/PolicyForm.tsx

```diff
@@ -16,6 +17,34 @@ import {
 import { FormState, FormValue } from "./types";
 import MultiItem from "./components/MultiItem";
 
+const textInput = (
+  value: string,
+  handleValueChange: (index: number, newValue: string) => void,
+  index: number,
+) => {
+  return (
+    <input
+      type="string"
+      class="font-semibold text-base text mr-2 border-2 border-black py-1 px-2 focus:outline-offset-2 focus:outline-2 focus:outline-yellow-400"
```

Comment:
Again not something from this change, but if we don't do the above comment, can we change this to `className` instead of `class`. `class` is a reserved keyword in JavaScript which is why we do `className` in react jsx.